server/store/sqlstore/store.go

```diff
@@ -730,6 +734,35 @@ func (s *SQLStore) UpdateSubscriptionExpiresOn(subscriptionID string, expiresOn
 	return nil
 }
 
+func (s *SQLStore) UpdateSubscriptionLastActivityAt(subscriptionID string, lastActivityAt time.Time) error {
+	query := s.getQueryBuilder().Update(subscriptionsTableName).Set("lastActivityAt", lastActivityAt.UnixMicro()).Where(sq.Eq{"subscriptionID": subscriptionID}, sq.Lt{"lastActivityAt": lastActivityAt.Add(-10 * time.Second).UnixMicro()})
+	_, err := query.Exec()
+	if err != nil {
+		return err
```

Comment:
For the global subscription, I expect this `UPDATE` to fail periodically due to concurrent writes. Propose we wrap this update in a `READ COMMITTED` transaction that will have Postgres seamlessly retry (re-evaluating the `WHERE` conditions). (Also, I'm separately proposing that we make this plugin Postgres only.)

Reply:
This shouldn't happen concurrently for the same subscription.

Reply:
Still, they are idempotent and 1 transaction per query, so I don't see any problem here. We can talk it later today.

Reply:
As discussed offline, the default transaction isolation level is already `READ COMMITTED`, so we're all good 🎉